### PR TITLE
Cache images with react-native-fast-image

### PR DESCRIPTION
**Pull request by tsturtz**
**What does this implement/fix? Explain your changes.**
Cache images with react-native-fast-image lib - installed npm package and replaced <Image /> with <FastImage /> imports and components

**Does this close any currently open issues?**
#78 

**Screenshots**
N/A

**Any other comments?**

**Where has this been tested?**
Diffs ;)

### CODE REVIEW

**Comment by troublediehard** [comment]
Thanks, but looks like you never run the project. Your work will make more sense after https://github.com/troublediehard/magento-react-native/issues/79, so I will wait for it.

**Comment by alexakasanjeev** [comment]
Let me test this branch, will update about status

File: src/components/account/OrderScreen.js [hunk 1]
```diff
@@ -1,10 +1,6 @@
 import React, { useEffect, useContext } from 'react';
-import {
-  View,
-  Image,
-  StyleSheet,
-  FlatList,
-} from 'react-native';
+import { View, StyleSheet, FlatList } from 'react-native';
+import { FastImage } from 'react-native-fast-image';
```
Comment (alexakasanjeev) [inline src/components/account/OrderScreen.js]: Please refer to the offical doumn

File: src/components/cart/CartListItem.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import React, { useContext } from 'react';
-import {
-  View, Image, TouchableOpacity, Alert,
-} from 'react-native';
+import { View, TouchableOpacity, Alert } from 'react-native';
+import { FastImage } from 'react-native-fast-image';
```
Comment (alexakasanjeev) [inline src/components/cart/CartListItem.js]: change named import to default import

File: src/components/catalog/ProductMedia.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React, { PureComponent } from 'react';
 import Swiper from 'react-native-swiper';
-import { View, Image } from 'react-native';
+import { View } from 'react-native';
+import { FastImage } from 'react-native-fast-image';
```
Comment (alexakasanjeev) [inline src/components/catalog/ProductMedia.js]: change named import to default import

File: src/components/common/ProductListItem.js [hunk 4]
```diff
@@ -1,10 +1,7 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import {
-  View,
-  Image,
-  TouchableOpacity,
-} from 'react-native';
+import { View, TouchableOpacity } from 'react-native';
+import { FastImage } from 'react-native-fast-image';
```
Comment (alexakasanjeev) [inline src/components/common/ProductListItem.js]: change named import to default import

File: src/components/home/FeaturedProductItem.js [hunk 5]
```diff
@@ -1,11 +1,7 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import {
-  StyleSheet,
-  TouchableOpacity,
-  View,
-  Image,
-} from 'react-native';
+import { StyleSheet, TouchableOpacity, View } from 'react-native';
+import { FastImage } from 'react-native-fast-image';
```
Comment (alexakasanjeev) [inline src/components/home/FeaturedProductItem.js]: change named import to default import

File: src/components/home/HomeSlider.js [hunk 6]
```diff
@@ -1,8 +1,7 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import {
-  View, StyleSheet, Image,
-} from 'react-native';
+import { View, StyleSheet } from 'react-native';
+import { FastImage } from 'react-native-fast-image';
```
Comment (alexakasanjeev) [inline src/components/home/HomeSlider.js]: change named import to default import

File: src/components/account/OrderScreen.js (hunk 1 above)
Comment (alexakasanjeev) [inline src/components/account/OrderScreen.js]: Please refer to the official documentation of [react-native-fast-image](https://github.com/DylanVann/react-native-fast-image#readme) 

Correct way of importing `FastImage` component is 
```
import FastImage from 'react-native-fast-image'
```

File: src/components/cart/CartListItem.js (hunk 2 above)
Comment (alexakasanjeev) [inline src/components/cart/CartListItem.js]: Change named import to default import

**Comment by alexakasanjeev** [comment]
@all-contributors please add tsturtz for code

**Pull request merged**